packages/components/src/globals/scss/vendor/@carbon/elements/scss/themes/_theme-maps.scss

```diff
@@ -172,7 +172,7 @@ $carbon--theme--g90: (
   focus: #ffffff,
   hover-primary: #0353e9,
   active-primary: #0530ad,
-  hover-primary-text: #97c1ff,
+  hover-primary-text: #054ada,
```

Comment:
Hey there, I'm not sure why this vendor file updated a token I did not touch, `hover-primary-text`? Unless that token needed to be updated? 🤔 Note that I ran `yarn build` inside the `components` package in order to add this new color to the vendor files...

Reply:
Ooo good catch, this might not be covered by our `sync` check for artifacts, let me dig in 👀

Reply:
Something confusing to me is that this value is `#054ada` in latest: https://github.com/carbon-design-system/carbon/blob/master/packages/components/src/globals/scss/vendor/%40carbon/elements/scss/themes/_theme-maps.scss#L47 🤔

Reply:
Oh whoa 😅 So this is correct & I should just keep it? Would you rather I fix it in a separate PR for history/documentation purposes?